### PR TITLE
Add simple unit tests

### PR DESCRIPTION
**Pull request by Davfon**
This pull request contributes to issue #6207, which is to add more unit tests or improve existing ones.

I added the trivial, but important, test case, that the output should remain unmodified in case the input was already valid.

Tests extended:

- CleanupUrlFormatterTest
- RemoveHyphenatedNewlinesFormatterTest
- RemoveNewlinesFormatterTest
- LowerCaseFormatterTest
- RemoveBracketsTest

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java [hunk 1]
```diff
@@ -29,6 +29,12 @@ void extractURLFormLink() {
                 formatter.format("away.php?to=http%3A%2F%2Fwikipedia.org&a=snippet"));
     }
 
+    @Test
+    void validURLUnmodified() {
+        // the caller has to pay attention that this does not happen
```
Comment (koppor) [inline src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java:34]: What should not happen? Maybe just remove the comment.

The call seems IMHO legal.

File: src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java [hunk 2]
```diff
@@ -29,6 +29,12 @@ void extractURLFormLink() {
                 formatter.format("away.php?to=http%3A%2F%2Fwikipedia.org&a=snippet"));
     }
 
+    @Test
+    void validURLUnmodified() {
```
Comment (koppor) [inline src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java:33]: We try to gradually migrate to Google's casing rules: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Thus, please introduce defined camel case

```suggestion
    void validUrlUnmodified() {
```

File: src/test/java/org/jabref/logic/formatter/casechanger/LowerCaseFormatterTest.java [hunk 3]
```diff
@@ -19,6 +19,7 @@ public void setUp() {
 
     @Test
     public void test() {
+        assertEquals("lower", formatter.format("lower"));
```
Comment (koppor) [inline src/test/java/org/jabref/logic/formatter/casechanger/LowerCaseFormatterTest.java:22]: Could you maybe change that to a paramterized test? We aim for having one assertion per test case. See also https://github.com/JabRef/jabref/issues/6207

grafik

**Review by koppor — CHANGES_REQUESTED** [review]

**Comment by Davfon** [comment]
I addressed the requested changes in the last commit, should be all good now.

File: src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java (hunk 1 above)
Comment (Davfon) [reply src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java]: That's on me, I copy-pasted and forgot to remove it.

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**